config/config.exs

```diff
@@ -2,8 +2,20 @@
 import Config
 
 # Configure fork
-# Available: :capella, :deneb
-config :lambda_ethereum_consensus, :fork, :capella
+# Available: "capella", "deneb"
+# Used only for testing
+fork_raw = System.get_env("FORK", "capella")
```

Comment:
this is at compile time?

Reply:
Locally, you'd need to recompile manually to change forks.